dallinger/frontend/templates/dashboard_monitor.html

```diff
@@ -88,10 +81,37 @@ <h5 class="card-title">{{ pane }}</h5>
                         </div>
                     {% endfor %}
                 </div>
+            </div>
+            <div class="collapse p-3 bg-light" id="config">
+                <h4>Config</h4>
+                <div class="custom-control custom-switch">
 
+                    <input type="checkbox" class="custom-control-input" id="auto_recruit"
```

Comment:
Maybe a Bootstrap 'switch' would be nice?

![image](https://user-images.githubusercontent.com/22684998/231524297-168bf795-4906-4ba6-a9df-8a8959993496.png)

Reply:
Done